### PR TITLE
Hackathon tutorial

### PR DESCRIPTION
**Pull request by Jimvin**
Fixes to the tutorial as some of the resources used have been moved or modified.

### CODE REVIEW

**Review by fhennig — CHANGES_REQUESTED** [review]
Just one minor thing basically

File: modules/tutorials/pages/end-to-end_data_pipeline_example.adoc [hunk 1]
```diff
@@ -59,19 +59,138 @@ NiFi is an ETL tool which will be used to model the dataflow of downloading and
 It will also be used to convert the file content from CSV to JSON.
 
 [source,bash]
-helm install --repo https://repo.stackable.tech/repository/helm-test nifi-operator nifi-operator --version=0.6.0-pr251
-
-NOTE: You need the PR version which has increased PVC sizes and heap memory. This will be merged soon.
-Apply the Kafka custom resource (https://github.com/stackabletech/kafka-operator/blob/main/examples/simple-kafka-cluster.yaml)
+helm install --repo https://repo.stackable.tech/repository/helm-dev nifi-operator nifi-operator --version=0.6.0-nightly
```
[the remark this answers is not in the document]
Comment (fhennig) [reply modules/tutorials/pages/end-to-end_data_pipeline_example.adoc:62]: So this is nothing to fix, just a comment :D

File: modules/tutorials/pages/end-to-end_data_pipeline_example.adoc [hunk 2]
```diff
@@ -59,19 +59,138 @@ NiFi is an ETL tool which will be used to model the dataflow of downloading and
...
 
 === Deploying Kafka and NiFi
 
-For the purposes of this use case, no special config is required, so you can simply deploy the examples straight from the Operator repositories.
-These files are designed to be self-contained, so for example the NiFi example also contains the necessary ZooKeeper cluster definition.
+To deploy Kafka and NiFi we can now apply our cluster configuration. We'll also need to deploy ZooKeeper, since both Kafka and NiFi depend on it. Run the following command in the console to deploy and configure all three services.
```
[the remark this answers is not in the document]
Comment (Jimvin) [reply modules/tutorials/pages/end-to-end_data_pipeline_example.adoc:66]: Done!

**Review by fhennig — APPROVED** [review]
LGTM!

**Pull request merged**